prime-dotnet-webapi/Sentry/SentryErrorReporter.cs

```diff
@@ -0,0 +1,40 @@
+using System;
+using System.Threading.Tasks;
+
+using Prime;
+using SharpRaven;
```

Comment:
Is `SharpRaven` being used?

Reply:
Yes, for creating the `_client` instance and for initializing it in lines 11 and 22 respectively.